docs/css/doc.css

```diff
@@ -139,6 +139,16 @@
   margin-bottom: var(--size-4);
 }
 
+/* Slider */
+.container_component .price_field {
+  width: 40%;
+}
+
+.container_component .price_field input {
+  width: 80%;
+  /* display: block; */
```

Comment:
Take care to cleanup unused commented code

Reply:
roger that